test/crash/src/test/java/io/narayana/lra/arquillian/LRACoordinatorRecoveryIT.java

```diff
@@ -247,7 +213,6 @@ public void lraCoordinatorRecoveryTwoLRAs(@ArquillianResource @OperateOnDeployme
         }
 
         lraClient.closeLRA(longLRA);
-        lraClient.closeLRA(shortLRA);
```

Comment:
Please can you mention why removing this call is definitely fine now?

Reply:
There was a mistake in the test: the waiting time (doWait) was not enough to trigger the shortLRA timeout and so it needed to be closed manually.
SHORT_TIMEOUT is 10.000
LRAListener.LRA_SHORT_TIMELIMIT is 5L
(LRAListener.LRA_SHORT_TIMELIMIT +1L ) * 1000 = 6000

Now the shortLRA is already closed at the end of the test. I think now it makes more sense.
BTW calling closeLRA to a cancelled LRA returns a 412 error response (precondition failed)

Reply:
This makes sense to me now. So if the LRA was cancelled (rather than cancelling) before these changes, this line would have caused a test failure.

Reply:
Yes, I believe so.